### PR TITLE
feat(pt-br-translation): claim-your-front-end-libraries-certificate

### PR DESCRIPTION
**Pull request by aferreira44**
#### Pre-Submission Checklist

- [X] Your pull request targets the `staging` branch of freeCodeCamp.
- [X] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [X] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [X] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [ ] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [X] Add new translation (feature adding new translations)

#### Checklist:

- [X] Tested changes locally.
- [X] Addressed currently open issue (replace XXXXX with an issue no in next line)

Closes freeCodeCamp/TranslationExpansion#45

#### Description

Brazilian Portuguese translation of Claim Your Front End Development Certificate

### CODE REVIEW

**Comment by raisedadead** [comment]
@aferreira44 awesome, can we have this peer reviewed? You can post on the forum or the chat room for assistance and interested people will get back to you. Post that we can get this merged.

**Comment by aferreira44** [comment]
@nmdoliveira and @samanthakem Can you review my PR ?

**Comment by nmdoliveira** [comment]
@aferreira44 I will review :wink:

File: seed/challenges/03-front-end-libraries/claim-your-front-end-libraries-certificate.json [hunk 1]
```diff
@@ -101,6 +101,35 @@
...
             ]
           ]
+        },
+        "pt-br": {
+          "title": "Solicite seu Certificado de Bibliotecas Front End",
+          "description": [
+            [
+              "//i.imgur.com/k8btNUB.jpg",
+              "Uma imagem do nosso Certificado de Bibliotecas Front End",
+              "Este desafio lhe dará seu certificado verificado de bibliotecas Front End. Antes de emitir o seu certificado, devemos verificar se você completou todos os nossos desafios de algoritmos básicos e intermediários e todos os nossos projetos básicos, intermediários e avançados de desenvolvimento. Você também deve aceitar nosso compromisso de honestidade acadêmica. Clique no botão abaixo para iniciar este processo.",
```
Comment (nmdoliveira) [inline seed/challenges/03-front-end-libraries/claim-your-front-end-libraries-certificate.json]: aqui ao invés de "devemos verificar se você completou..." eu acho que fica melhor "precisamos verificar que você completou..." e também acho que fica legal capitalizar o nome do compromisso igual ao original, "Compromisso de Honestidade Acadêmica"

File: seed/challenges/03-front-end-libraries/claim-your-front-end-libraries-certificate.json [hunk 2]
```diff
@@ -101,6 +101,35 @@
...
+              ""
+            ],
+            [
+              "//i.imgur.com/uLPsUko.jpg",
+              "A definição de plágio: Plágio (substantivo) - copiar o trabalho de outra pessoa e apresentá-lo como seu sem creditar o autor",
+              "Ao clicar abaixo, você promete que todo o seu código enviado A) foi escrito por você ou o seu par pessoalmente, ou B) vem de bibliotecas de código aberto como o jQuery ou C) foi claramente atribuído a seus autores originais. Você também nos dá permissão para auditar suas soluções de desafios e revogar o seu certificado se descobrirmos evidências de plágio.",
```
Comment (nmdoliveira) [inline seed/challenges/03-front-end-libraries/claim-your-front-end-libraries-certificate.json]: ao invés de "você promete que todo o seu código..." acho que fica melhor "você declara que todo o seu código..."

File: seed/challenges/03-front-end-libraries/claim-your-front-end-libraries-certificate.json [hunk 3]
```diff
@@ -101,6 +101,35 @@
...
+            [
+              "//i.imgur.com/UedoV2G.jpg",
+              "Uma imagem do texto \"Requisitos do Certificado de Desenvolvimento Front-End\"",
+              "Vamos confirmar que você completou todos os nossos desafios de algoritmos básicos e intermediários e todos os nossos projetos básicos, intermediários e avançados de desenvolvimento. Clique no botão abaixo para verificar.",
+              "#"
+            ],
+            [
+              "//i.imgur.com/Q5Za9U6.jpg",
+              "Uma imagem da palavra \"Parabéns\"",
+              "Parabéns! Adicionamos o seu Certificado de Bibliotecas Front End à sua página de portfólio. A menos que você opte por esconder suas soluções, este certificado permanecerá publicamente visível e verificável",
```
Comment (nmdoliveira) [inline seed/challenges/03-front-end-libraries/claim-your-front-end-libraries-certificate.json]: ponto final aqui haha

**Review by nmdoliveira — COMMENTED** [review]
just a few suggestions

**Comment by aferreira44** [comment]
Thanks @nmdoliveira for the review. I accepted your suggestions. 👌 👍 

@raisedadead PR reviewed.

**Comment by raisedadead** [comment]
Awesome! Thanks @aferreira44 and @nmdoliveira !

**Pull request merged**